inspec.gemspec

```diff
@@ -38,4 +38,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'sslshake', '~> 1'
   spec.add_dependency 'parallel', '~> 1.9'
   spec.add_dependency 'rspec_junit_formatter', '~> 0.2.3'
+  spec.add_dependency 'requests', '~> 1.0.0'
```

Comment:
@chris-rock yes it isn't well maintained but it works like a charm.
I try to use native Net::HTTP but it finally was quite the same as `requests`... with less stability.
I also try to find another lib without success.

`requests` wasn't a wonderful choice but a pragmatic one. :/